src/patternfly/components/Table/table.scss

```diff
@@ -313,6 +314,7 @@
 
   > * {
     margin-top: var(--pf-c-table__check--input--MarginTop);
+    font-size: var(--pf-c-table__check--input--FontSize);
```

Comment:
wdyt about redefining `--pf-c-table-cell--FontSize` in `.pf-c-table__check` instead? So move this up to the `.pf-c-table__check` selector and change to `--pf-c-table-cell--FontSize: var(--pf-c-table__check--input--FontSize);`